docs/guide/03-styling-templates.md

```diff
@@ -15,6 +15,53 @@ To help with dynamic styling, lit-html provides two directives for manipulating
 *   [`classMap`](template-reference#classmap) sets classes on an element based on the properties of an object.
 *   [`styleMap`](template-reference#stylemap) sets the styles on an element based on a map of style properties and values.
 
+## Inline styles with styleMap {#stylemap}
+
+You can use the `styleMap` directive to set inline styles on an element in the template.
+
+```js
+import {html} from 'lit-html';
+import {styleMap} from 'lit-html/directives/style-map.js';
+
+const normalStyles = {};
+const highlightStyles = { color: 'white', backgroundColor: 'red'};
```

Comment:
I think the canonical example for this `styleMap` is a single literal with dynamic values. That would remove the ternary below and simplify the template.

Reply:
Sure... Not sure exactly what you mean, though. 

Are you thinking of a literal inside the `myTemplate` function here, like:

```
const styles =  { color: currentColor; background-color: currentBackground};
```

Reply:
Yeah, though I might do something like move the ternary into one of the expressions to drive home that it's dynamic:

```js
const styles =  {
  color: currentColor;
  background-color: highlight ? 'yellow' : currentBackground;
};
```